.github/workflows/build.yml

```diff
@@ -145,7 +157,7 @@ jobs:
       - name: Set Up Env
         run: make env
       - name: Run Tests
-        run: make test
+        run: make test args=tests/vision/gpu_tests
```

Comment:
why did you add to make `test-tabular-only` if you can do it with args like here?

Reply:
main reason is it is not installing vision